### ISSUE TITLE
import errors

### ISSUE THREAD

**Issue by hunterowens** [CityofSantaMonica/mds-provider#34]
If I run 

```
import mds
mds.providers` 
```

currently raises a `AttributeError`. 

However, 

`from mds import providers` works fine. I suspect this is some `__init__.py` fu that I need to fix.

**Comment by thekaveman** [comment]
Hoping that #36 can at least rule out the possibility of some weird version mismatch or pip caching or something.

**Comment by thekaveman** [comment]
I think since `__init__.py` doesn't import `mds.providers`, then `providers` doesn't exist as a name under `mds`.

But `mds.providers` exists as a name itself -- confusing.

Solution might be to `import` the rest of the package in [`mds/__init__.py`](https://github.com/CityofSantaMonica/mds-provider/blob/master/mds/__init__.py)?

```python
import mds.api
import mds.db
import mds.fake
import mds.json
import mds.providers
import mds.schema
```

**Comment by thekaveman** [comment]
@hunterowens this should all be fixed once #71 is merged via #40. 

Check out [this commit](https://github.com/CityofSantaMonica/mds-provider/pull/71/commits/6b6e26c5d870c8990c9ccb8b662c0abc55b0d09f), which cleaned up all of the imports.
